.github/workflows/main.yml

```diff
@@ -18,7 +18,9 @@ jobs:
       - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: 21
+          java-version: |
+            22
```

Comment:
This might not work, as we pinned the JDK by toolchain

https://github.com/melix/jmh-gradle-plugin/blob/e70fb4ea624c534c0ffdf384f453935fd1e9b78a/build.gradle.kts#L54

Reply:
Yea - I (kinda) had to change the test-project for `ProjectWithFeaturePreviewSpec` to use 22, as 15 is no longer available (easier to setup 22 locally than 15, for example via sdkman).

Reply:
Just need 22 "somewhere". Build is still running on 21

Reply:
FYI, I did something related 

https://github.com/melix/japicmp-gradle-plugin/blob/4a68c4a3c780009c9fbcf06cb0058fa176b91615/build-logic/src/main/kotlin/me.champeau.convention-testing.gradle.kts#L23-L63

Reply:
But isn't this different from how `ProjectWithFeaturePreviewSpec`'s test project works?

Reply:
Just see.